src/cryptoadvance/specter/specter.py

```diff
@@ -124,9 +126,9 @@ def __init__(self, data_folder="./data", config={}):
             self.check(check_all=True)
         except Exception as e:
             logger.error(e)
-        self.checker = Checker(lambda: self.check(check_all=True))
+        self.checker = Checker(lambda: self.check(check_all=True), desc="health")
         self.checker.start()
-        self.price_checker = Checker(lambda: update_price(self, self.user))
+        self.price_checker = Checker(lambda: update_price(self, self.user), "price")
```

Comment:
```suggestion
        self.price_checker = Checker(lambda: update_price(self, self.user), desc="price")
```
Right now the "price" is passed as the period which breaks the checker,  should be `desc="price"` instead